getting_started/bootstrap.rst

```diff
@@ -57,11 +57,31 @@ Start by creating a ``terraform.tfvars`` file. There is a template that you can
 - **cluster_prefix**: every resource in your tenancy will be named with this prefix
 - **KuberNow_image**: name of the image that you previously imported using Ansible
 - **ssh_key**: path to your public ssh-key to be used (for ssh node access)
-- **floating_ip_pool**: a floating IP pool name
+- **floating_ip_pool**: a floating IP pool label
 - **external_network_uuid**: the uuid of the external network in the OpenStack tenancy
 - **dns_nameservers**: (optional, only needed if you want to use other dns-servers than default 8.8.8.8 and 8.8.4.4)
 - **kubeadm_token**: a token that will be used by kubeadm, to bootstrap Kubernetes. You can run generate_kubetoken.sh to create a valid one.
 
```

Comment:
This additional info should go in troubleshoot, many people know this things, and it becomes slower to read IMO. However, it would be nice to have a cross-reference that brings the reader to the correct place where we can find this info.